common_drivers/xcvr_core/xcvr_core.c

```diff
@@ -228,73 +228,93 @@ int32_t xcvr_setup(xcvr_core *core)
 	uint32_t rx_out_div;
 	uint32_t tx_out_div;
 	uint32_t ln_rate_kbps;
-	xcvr_cpll cpll_config;
-	xcvr_qpll qpll_config;
+	struct xilinx_xcvr_cpll_config cpll_config;
```

Comment:
I still feel that this commit is doing too much. Even if this is not the case, you need to convince the reviewers by describing what you are actually doing in the commit message. What does adding support for Ultrascale and Ultrascale+ imply? What are the differences from previously supported architectures?